rockcraft/models/project.py

```diff
@@ -80,9 +80,30 @@
     ),
 ]
 
-BaseT = Literal["bare", "ubuntu@20.04", "ubuntu@22.04", "ubuntu@24.04"]
+BaseT = Literal[
+    "bare",
+    "ubuntu@20.04",
+    "ubuntu@22.04",
+    "ubuntu@24.04",
+    "ubuntu@25.04",
```

Comment:
I don't think we need to support `25.04` (but I'm fine with leaving it in for testing purposes while it's a prototype)